docs/installation/prepare/microcontrollers/_index.md

```diff
@@ -26,361 +26,245 @@ Recommended configuration: 384kB Ram + 8MB SPIRAM + 4MB Flash
+Use [the micro-RDK template](https://github.com/viamrobotics/micro-rdk-template.git) to create a new micro-RDK project to upload to your ESP32 by running:
 
-   ![Create remote menu](../../img/microcontroller/create-remote.png)
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+cargo generate --git https://github.com/viamrobotics/micro-rdk-template.git
+```
 
-3. Paste the copied `JSON` config into the configuration field.
+If you would like, you can use `mkdir` to initialize a new repository in the directory you created by running `cargo-generate`, to track any changes you make to the generated project.
 
-   ![Empty configuration field](../../img/microcontroller/config-field.png)
+You will be prompted to paste your Viam robot configuration information (`viam.json`) into the terminal.
 
-   Ensure that the controlling robot is live in [the Viam app](https://app.viam.com).
-   If it is, the ESP32-backed robot is now programmatically available.
+To obtain this, navigate to [the Viam app](https://app.viam.com).
+Click the **Copy viam-server config** button on the right side of the **Setup** tab of your robot.
+The Micro-RDK uses the config for communication with the Viam app.
+Paste this into your terminal.
 
-## Tips
+{{< alert title="Caution" color="caution" >}}
 
-When developing, you can use the QEMU emulator instead of flashing the ESP32 board.
-Follow these instructions to install and build the emulator.
+All of the generated files should be safe to commit as a project on Github, with the exception of `viam.json`, since it contains a secret key.
 
-### Install Espressif's QEMU ESP32 Emulator
+{{% /alert %}}
 
-Espressif maintains a good [QEMU emulator](https://github.com/espressif/qemu) supporting the ESP32.
+### Upload and Connect to your ESP32 Board
 
 {{< tabs >}}
-{{% tab name="macOS" %}}
-
-Run the following command to install the QEMU ESP32 Emulator:
+{{% tab name="Using Canon" %}}
 
 ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
-git clone https://github.com/espressif/qemu
-cd qemu
-./configure --target-list=xtensa-softmmu \
-    --enable-gcrypt \
-    --enable-debug --enable-sanitizers \
-    --disable-strip --disable-user \
-    --disable-capstone --disable-vnc \
-    --disable-sdl --disable-gtk --extra-cflags="-I/opt/homebrew/Cellar/libgcrypt/1.10.1/include -I/opt/homebrew//include/"
-cd build && ninja
+canon bash -lc "make build-esp32-bin"
+make flash-esp32-bin
 ```
 
 {{% /tab %}}
-{{% tab name="Linux" %}}
-
-On Ubuntu or Debian, first make sure you have the `libgcrypt` library and headers installed by running the following command:
-
-```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
-sudo apt-get install libgcrypt20 libgcrypt20-dev
-```
+{{% tab name="Local environment" %}}
 
-Then, run the following command to install QEMU:
+Make sure you have sourced `. ~/dev/esp/export-rs.sh` and `. ~/dev/esp/esp-idf/export.sh` prior running the following command

```

Comment:
```suggestion
Make sure you have sourced `. ~/dev/esp/export-rs.sh` and `. ~/dev/esp/esp-idf/export.sh` before running the following command: 
```
"sourced" could be confusing to some but I can't think of a better way to phrase

Reply:
we can use run I agree it sounds better